addons/langfuse/templates/_helpers.tpl

```diff
@@ -76,6 +76,14 @@ Create the name of the secret for postgresql if we use an external database
 {{- printf "%s-postgresql" (include "langfuse.fullname" .) -}}
 {{- end }}
 
+{{/*
+Create the name of the secret for additional sensitive information
+*/}}
+{{- define "langfuse.additionalSecretName" -}}
```

Comment:
can we get a better name than additional secret name? like, `encryptionKeySecretName`?

Reply:
My thinking is we can use this for more variables than just that. It doesn't make much sense to create another secret for each sensitive variable. I am keeping the other two because they had it in the original helm chart.